DataFormats/Common/interface/DataFrameContainer.h

```diff
@@ -126,17 +126,18 @@ namespace edm {
 
     const_IterPair pair(size_t i) const { return const_IterPair(m_ids.begin() + i, m_data.begin() + i * m_stride); }
 
-    DataFrame operator[](size_t i) { return DataFrame(*this, i); }
+    DataFrame operator[](size_t i) { return DataFrame(*this, static_cast<DataFrame::size_type>(i)); }
```

Comment:
In these cases, wouldn't it be better to change the function parameter type to match what is expected ?
That is,
```suggestion
    DataFrame operator[](DataFrame::size_type i) { return DataFrame(*this, i); }
```
?